### ISSUE TITLE
[suggestion] Move to postfix .inline

### ISSUE THREAD

**Issue by sbeckeriv** [Stranger6667/css-inline#89]
Dearest Maintainer,

Neat project. I hope to use it soon. I was playing with the command line tool and was confused by the error messages.
```
 time ./target/release/css-inline  --remove-style-tags --load-remote-stylesheets ~/trash/1MB.html
/home/becker/trash/1MB.html: FAILURE (No such file or directory (os error 2))
```

My file is there. what is not there is inlined./home/becker/trash/1MB.html. If you move to postfix it will work for absolute and relative paths. 

Dictated but not reviewed.
Becker

**Comment by Stranger6667** [comment]
Hello @sbeckeriv 

Good catch! I definitely overlooked this behavior - prefix should only be applied to the filename itself :)

**Comment by Stranger6667** [comment]
The fix was released in `0.6.0`! :) Let me know if it solves the problem for you
